game/scripts/npc/abilities/templar_assassin_psi_blades.txt

```diff
@@ -23,7 +23,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_attack_range"                              "60 120 180 240 300 360"
+        "bonus_attack_range"                              "60 120 180 240 320 400" // OAA
```

Comment:
TA doesn't need a buff, she's amazing. At the very least, she doesn't need a buff while the phase boots interactions are still in the game.